### PR TITLE
Vignette

### PR DESCRIPTION
**Pull request by kendalynnm**
This is (imho) off to a strong start. I'll continue working on this today, filling in the "Under the hood" bits. When we meet tomorrow it would be useful to generate some visualizations for the two alternative-parameter optimizations listed in the TOC.

### CODE REVIEW

**Comment by bpbond** [comment]
1. I agree this is off to a strong start
2. What lovely LaTeX equations!
3. There are some spots perhaps make a little less academic-y and a little more vignette-y
4. Let's not use `dplyr` in the vignette; that introduces a whole new package dependency that we don't want (and that's why the checks are currently failing)
5. Let's talk about a `pdr_optimize_tidy()` function
6. Code note: should get message about frac_k and frac_P lookups
7. Move `Morris2023` visualization up top?

Overall LOOKS GREAT!

**Comment by kendalynnm** [comment]
@bpbond, I imported ggplot2 and tibble as dependencies using use_import_from(). Look good?

**Comment by bpbond** [comment]
@kendalynnm 

You'll want ggplot2 to be in "Suggests:", since it's only needed for the vignette, not using the package.

As for tibble...I'd suggest putting it into Suggests as well, for the same reason, and then we can discuss how much it's integrated into the main package.

File: vignettes/Intro_PoolDilutionR.Rmd [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+   pdr_optimize_tidy(time = samples$time_days,
+                     m = samples$cal12CH4ml + samples$cal13CH4ml,
+                     n = samples$cal13CH4ml,
+                     P = 0.1,
+                     pool = "CH4",
+                     m_prec = 0.001,
+                     ap_prec = 1,
+                     include_progress = FALSE)}
+   )
+all_results <- do.call(rbind, all_results)
```
Comment (kendalynnm) [inline vignettes/Intro_PoolDilutionR.Rmd:132]: Here I bring together the results from lapply, the sample id's are carried through, but it's a little clunky. Ideas for improving the tidy output? @bpbond

Comment (bpbond) [reply vignettes/Intro_PoolDilutionR.Rmd:132]: I don't have a cleaner way to do it without using `dplyr`.

**Comment by kendalynnm** [comment]
@bpbond, I believe this fail is a github problem, not a Kendal problem. If you agree, then we are ready to merge!!!!!

**Comment by bpbond** [comment]
@kendalynnm 👍  I agree, not our problem. Merge away! Woot!

**Review by bpbond — APPROVED** [review]
I didn't look at the vignette but can do that in a separate PR if useful.

**Pull request merged**